docs/contributors/code/coding-guidelines.md

```diff
@@ -176,9 +176,11 @@ if ( IS_GUTENBERG_PLUGIN ) {
 }
 ```
 
+The experimental APIs that already are publicly available via `window.wp` should remain accessible as removing them would break the existing plugins and themes.
```

Comment:
I'm not sure I agree with this personally. I think for the time being, we should continue with our current policies that we've been using for some time now. I think we can deprecate these APIs and remove them once we know they don't have any impact:

 - Deprecating
 - Giving time,
 - On timeout, checking usage and remove

Reply:
Hm yes – good note. I updated the document.